docs/mkdocs.yml

```diff
@@ -23,6 +23,7 @@ nav:
         - app set: 'cmd/aad/app/app-set.md'
         - app role add: 'cmd/aad/app/app-role-add.md'
         - app role list: 'cmd/aad/app/app-role-list.md'
+        - app role delete: 'cmd/aad/app/app-role-delete.md'
```

Comment:
Let's keep the TOC sorted alphabetically to make it easier for users to find the right command.